asteroid-launcher.pro

```diff
@@ -46,5 +46,7 @@ target.path = /usr/bin
 
 watchfaces.path = /usr/share/asteroid-launcher/watchfaces
 watchfaces.files =  watchfaces/*
+watchfaces-img.path = /usr/share/asteroid-launcher/watchfaces-img
+watchfaces-img.files =  watchfaces-img/*
 
 INSTALLS = target watchfaces
```

Comment:
This probably lacks a watchfaces-img

Reply:
could you eleborate?
Is the actual path missing or is this code faulty?

Reply:
The INSTALLS line points to the "targets" that should be installed.
It currently installs "target" which is the asteroid-launcher binary, to be installed in /usr/bin and to "watchfaces" which is the set of watchfaces for /usr/share/asteroid-launcher/watchfaces.
What you did is creating a new installable target named "watchfaces-img" but as long as you don't add it to the INSTALLS variable, it won't actually be installed